common/src/main/resources/assets/veil/lang/de_de.json

```diff
@@ -0,0 +1,178 @@
+{
+  "key.veil.editor": "Editor umschalten",
+  "key.categories.veil": "Schleier",

```

Comment:
The mod name is identical to the category name and therefore shouldn't be translated.